packages/commonwealth/client/scripts/views/pages/snapshot_proposals/snapshot_proposal_card.tsx

```diff
@@ -21,7 +21,7 @@ export const SnapshotProposalCard = (props: SnapshotProposalCardProps) => {
   const navigate = useCommonNavigate();
 
   const symbol = app.snapshot.space.symbol.toLowerCase();
-  const proposalLink = `/${symbol}/snapshot/${snapshotId}/${proposal.id}`;
+  const proposalLink = `/snapshot/${snapshotId}/${proposal.id}`;
```

Comment:
Choose community> Go to Snapshot> Active/Ended proposal> 
**Click Proposal for detail**
![image](https://user-images.githubusercontent.com/4791635/231202458-ddcf9b83-4a86-47ee-b5ee-c4d6e03dd187.png)
 
<img width="1026" alt="01_snapshot" src="https://user-images.githubusercontent.com/4791635/231202119-01afa35d-25cc-4a88-854b-78befd49aeef.png">

Fix:
removed /${symbol} from link